Initial commit

SkorpionNL opened the pull request:
Added On/Off plug-in unit as another Smart Plug type
New Smart plug Icon
Fixed some typo's
Added support for:
- Aqara Wireless Button (WXKG11LM)
- Ikea Symfonisk Sound Controller

SkorpionNL committed:
Initial commit

Added On/Off plug-in unit as another Smart Plug type
New Smart plug Icon
Fixed some typo's
Added support for:
- Aqara Wireless Button
- Ikea Symfonisk Sound Controller

notabene00 reviewed (changes requested):
For each`x_button_press` trigger we can specify multiple `driver_id`s
`"filter": "driver_id=mi-button|aqara-button|symfonisk-remote"` for example
It would be nicer
 
And, I think, version must be `0.6.0`
in readme and app.json

First number changes when we have breaking changes
Second - when new devices or big improvements
And third - some small changes or bug fixes

SkorpionNL committed:
Requested changes from PR master branch

- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.

SkorpionNL committed:
Revert "Requested changes from PR master branch"

This reverts commit 425273483c3fffdc4d8d97607bce81a16cbeb9bc.

SkorpionNL committed:
Requested changes from PR master branch

- Version bump to 0.6.0
- Combine filters for Mi button, Aqara button & Symfonisk remote for 1-4x button presses in app.json.
- Add check for reachable state for sensors.

SkorpionNL committed:
Merge branch 'master' of https://github.com/SkorpionNL/HomeyCONZ
